modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -30,22 +30,24 @@ $ rosa create admin --cluster=<cluster_name>
 W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
 I: Admin account has been added to cluster 'cluster_name'. It may take up to a minute for the account to become active.
 I: To login, run the following command:
-oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \
+oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \ #<1>
 --username cluster-admin \
 --password FWGYL-2mkJI-3ZTTZ-rINns
 ----
+<1> For {hcp-title-first} cluster, the port number should be `443`.
```

Comment:
Should this read as "For {hcp-title-first} clusters" or "For a {hcp-title-first} cluster"?

Reply:
> Should this read as "For {hcp-title-first} clusters" or "For a {hcp-title-first} cluster"?
"For a {hcp-title-first} cluster" looks good to me for here and future instances for this PR. I will make the updates accordingly for this PR.

> Also notice sometimes it's {hcp-title-first} and sometimes it's {hcp-title}...which I now see is because {hcp-title-first} is the first mention so it's the full name, and {hcp-title} is a subsequent mention.
Yes, exactly.